framework/include/mesh/MooseMesh.h

```diff
@@ -1430,6 +1430,15 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
   /// Return displace node list by side list boolean
   bool getDisplaceNodeListBySideList() { return _displace_node_list_by_side_list; }
 
+  /// Set incomplete interface pairs flag
+  void setHasIncompleteInterfacePairs(bool has_incomplete)
+  {
+    _has_incomplete_interface_pairs = has_incomplete;
+  };
```

Comment:
prefer that multi-line methods be outside the class definition. You can put it `inline` later in this file. Also remove semicolon at end of method definition